TECH.md

```diff
@@ -38,25 +38,29 @@ Reactive contract's static subscriptions are configured by calling the `subscrib
     }
 ```
 
-Reactive contracts can change their subscription dynamically by using callbacks to Reactive Network instances (as opposed to ReactVM) of themselves, which can, in turn, call the system contract to effect the appropriate changes to subscriptions.
+Reactive contracts can change their subscription dynamically by using callbacks to Reactive Network instances (as opposed to ReactVM) of themselves, which can, in turn, call the system contract to affect the appropriate changes to subscriptions.
```

Comment:
This change is incorrect, "effect" is not a typo here. Please revert.

https://en.wiktionary.org/wiki/effect#Verb